### PR TITLE
xclip: new package at 0.13

### PR DESCRIPTION
**Pull request by nikotidar**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/xclip/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./bootstrap
```
Comment (dylanaraps) [inline community/xclip/build]: Change this to `autoreconf -i`.

File: community/xclip/sources [hunk 2]
```diff
@@ -0,0 +1 @@
+https://github.com/astrand/xclip/archive/0.13.zip
```
Comment (dylanaraps) [inline community/xclip/sources]: Change source to: https://github.com/astrand/xclip/archive/0.13.tar.gz

File: community/xclip/version [hunk 3]
```diff
@@ -0,0 +1 @@
+0.13
```
Comment (dylanaraps) [inline community/xclip/version]: Missing `rel`.

**Comment by dylanaraps** [comment]
This package is missing dependencies:

```sh
autoconf make
automake make
libX11
libXt
libXmu
```

**Comment by dylanaraps** [comment]
Have you checked to see if this builds?

The package manager should have refused to build it with the missing `rel` in `version` and `zip` decompression isn't automatic...

File: community/xclip/build [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-./bootstrap
+autoreconf -i \
```
Comment (dylanaraps) [inline community/xclip/build]: Remove this backslash `\`.

File: community/xclip/version (hunk 3 above)
Comment (dylanaraps) [reply community/xclip/version]: (`0.13 1`)

File: community/xclip/build (hunk 4 above)
Comment (nikotidar) [reply community/xclip/build]: Okay, done.

File: community/xclip/version (hunk 3 above)
Comment (nikotidar) [reply community/xclip/version]: I forgot about that, sorry!
Seems to be fixed now.

**Comment by dylanaraps** [comment]
Does it build now?

**Comment by nikotidar** [comment]
> Does it build now?

current

Yes, its fixed.

**Comment by dylanaraps** [comment]
Thanks :+1:

**Comment by nikotidar** [comment]
> Thanks +1

Anytime, dylan! +1

**Pull request merged**